cmd/bootstrap/transit/cmd/pull.go

```diff
@@ -55,6 +54,15 @@ func pull(cmd *cobra.Command, args []string) {
 	// create new bucket instance with Flow Bucket name
 	bucket := gcs.NewGoogleBucket(flagBucketName)
 
+	// bump up the timeout for an execution node if it has not been explicitly set since downloading
+	// root.checkpoint takes a long time
+	if role == flow.RoleExecution && !cmd.Flags().Lookup("timeout").Changed {
```

Comment:
some EN node operators have had to specify `--timeout` explicitly. This is to just make it easier for them and set the default timeout for an execution node to be long enough to accommodate downloading of the large root.checkpoint files.